core_debug.tex

```diff
@@ -54,9 +54,14 @@ \section{Debug Mode} \label{debugmode}
 \end{steps}
 
 \begin{commentary}
-    In general, the debugger is expected to be able to simulate all the effects of \FcsrMcontrolMprv.
-    The exception is the case of Sv32 systems, which need \FcsrMcontrolMprv functionality in order to access
-    34-bit physical addresses. Other systems are likely to tie \FcsrDcsrMprven to 0.
+    When mprven=1, the external debugger can set MPRV and MPP appropriately to
```

Comment:
mprven -> \FcsrDcsrMprven

Reply:
Oops.  I keep forgetting to do that.  Fixed now.